examples/k3s/README.md

```diff
@@ -16,22 +31,35 @@ Or they all can be run with `make`:
 ```bash
 $ make
 ```
+K3S binary is already in the instances.
 
-You can then shell into any node, and check the status of K3S.
+### Upload K3S starter script and flux job submit script to ALL the nodes
 
 ```bash
-$ ssh -o 'IdentitiesOnly yes' -i "mykey.pem" rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
+$ scp -i "mykey.pem" k3s_starter.sh rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
+$ scp -i "mykey.pem" k3s_cleanup.sh rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
+$ scp -i "mykey.pem" k3s_agent_cleanup.sh rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
+$ scp -i "mykey.pem" flux_batch_job.sh rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
 ```
 
-Check the cluster status, the overlay status, and try running a job:
+### Note: the k3s deployment script (k3s_starter.sh) assume the clean up scripts are in the user home directory.
+
+You can then shell into any node, and submit flux jobs.
+
+```bash
+$ ssh -o 'IdentitiesOnly yes' -i "mykey.pem" rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
+```
 
+### Now, Run flux job that will start K3S, and will run your workload
+Be sure to change k3s secret value, number of instances, and any modifications!
+The below command runs a job with three nodes
 ```bash
-$ kubectl get nodes
+$ flux batch -N 3 --error k3s_installation.out --output k3s_installation.out flux_batch_job.sh "k3s_secret_token"
```

Comment:
Is there meaningful output in this file? I ask because when I've used batch (and tried to set the output file) I saw in the logs just a reference to other output files (the ones in the batch script). I wonder if we should bring this up with flux devs to have some ability to specify either an output directory (for those files) or stream to one file (could be messy, which is why they probably don't do it).

Reply:
Yes, some of the outputs from the batch script are written here. For example, output of the flux exec will be printed here. Inside the batch script, if you run flux submit, that job output will be written other places (you defined or system defined). So, Inside the script, I'm using another file to log k3s installation related outputs. so, when the `flux submit` start, you can see the logs specific from the `flux submit` job.

Reply:
I think we could improve upon this in flux, but that's a discussion for another place. Thank you for the details.